content/en/docs/tasks/security/authn-policy/index.md

```diff
@@ -383,6 +252,9 @@ command terminated with exit code 56
 If we have more services in namespace `bar`, we should see traffic to them won't be affected. Instead of adding more services to demonstrate this behavior,
 we edit the policy slightly to apply on a specific port:
 
+
+<!-- What's this part for? I don't think we have a port 1234 for httpbin service. -->
```

Comment:
@diemtvu you know about this? What's 1234 port for? Just an example for non existing port?

Reply:
It shows port-specific policy won't affect other ports. I know, it's not a great example :). Feel free to change if you want